screentest.ui

```diff
@@ -157,6 +157,16 @@
                 <signal name="activate" handler="on_mode_change"/>
               </object>
             </child>
+            <child>
+              <object class="GtkRadioMenuItem" id="bright_pixels">
+                <property name="visible">True</property>
+                <property name="label" translatable="yes">Bright Pixels</property>
```

Comment:
I wonder if "Solid color" would be the better label? Not sure about this myself :smile:

Reply:
The Name is no matter. "Solid Color Cycle RGBWB". What ever ;)